_posts/2017/2017-02-02-programming-beyond-the-comfort-zone-javascript-elixir.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "Programming beyond the comfort zone: JavaScript |> Elixir"
+date: "2017-02-02 07:00:00"
+image: blog-header/drink.jpg
+categories: tech-stories
+authors: ["Erik"]
+---
+
+I'm a JavaScript Engineer.
+I write JavaScript.
+I really like writing JavaScript and I feel very comfortable doing so.
+In the JavaScript world, there are a gazillion web frameworks available and every week there are more coming.
+Besides the usual suspects like `express` and `hapi`, there are a lot of things that are fun to play with like zeit's `micro`.
+Although having some different concepts, they all have something in common: They're (of course) JavaScript frameworks.
+Now, if you want to learn more about the HTTP stack or the request and response cycle, you should definitely check out a lot of different frameworks since different concepts give you different insights and perspectives of the inner workings.
+
+So why should you start programming in another language?
+
+Let's take a step back from the frameworks and look at the language itself.
+JavaScript claims to be both, object-oriented (with prototypal inheritance) and functional.
+And since it's both, it's neither nor to a satisfying extend.
```

Comment:
@erikmueller I've never seen this constellation in English: "neither nor"
Should be something like "neither this nor that". But what would it be?